.github/actions/build/action.yml

```diff
@@ -1,10 +1,14 @@
 name: build
 description: Build all packages
 inputs:
+  REF_NAME:
+    description: 'Git ref name'
+    required: true
+    default: 'test'
   NODE_ENV:
     description: 'Node environment'
     required: false
-    default: 'staging'
+    default: 'test'
```

Comment:
I think we should change these configs as well to include `test`. Otherwise it may not minify the bundle.
<img width="714" alt="image" src="https://github.com/binary-com/deriv-app/assets/56330681/a0cf8497-21cb-4fe0-9830-420e1723efd8">

Reply:
I just added it to all of the webpack configs in the app.